src/particles/wakefields/HandleWakefield.H

```diff
@@ -0,0 +1,150 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Alex Bojanich, Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef HANDLE_WAKEFIELD_H
+#define HANDLE_WAKEFIELD_H
+
+#include "particles/ImpactXParticleContainer.H"
+#include "ChargeBinning.H"
+#include "CSRBendElement.H"
+#include "WakeConvolution.H"
+#include "WakePush.H"
+
+#include <cmath>
+#ifndef ImpactX_USE_FFT
+#include <stdexcept>
+#endif
+#include <vector>
+
+
+namespace impactx::particles::wakefields
+{
+    /** Function to handle CSR bending process including charge deposition,
+     * mean transverse position calculation, wakefield generation, and convolution.
+     *
+     * @param[in] particle_container the particle species container
+     * @param[in] element_variant variant type of the lattice element
+     * @param[in] slice_ds slice spacing along s
+     * @param[in] print_wakefield for debugging: print the wakefield to convoluted_wakefield.txt
+     */
+    template <typename T_Element>
+    void HandleWakefield (
+        impactx::ImpactXParticleContainer& particle_container,
+        T_Element const& element_variant,
+        amrex::Real slice_ds,
+        bool print_wakefield = true
```

Comment:
```suggestion
        bool print_wakefield = false
```
Let's set this to 'false' by default.

Reply:
Phew, good catch!